.github/workflows/triage_add_stale_issue_to_board.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Add stale issue/PR to Triage Board'
+on:
+  issues:
+    types: [labeled]
+  pull_request:
+    types: [labeled]
```

Comment:
Do we care about stale PRs? IMO we should go through & just close stale PRs but leave the branch.

Reply:
we want to be aware of them.  I don't see a reason for keeping them.  We will only be closing the PR and deleting the branch for the time being.